src/v/cluster/health_monitor_backend.cc

```diff
@@ -18,6 +18,7 @@
 #include "cluster/logger.h"
```

Comment:
Please add some information to the commit message what is the rationale behind this change